src/HypothesisClient/ApiClient/ApiClientTrait.php

```diff
@@ -1,22 +1,38 @@
 <?php
 
-namespace eLife\HypothesisClient;
+namespace eLife\HypothesisClient\ApiClient;
 
+use eLife\HypothesisClient\Credentials\Credentials;
+use eLife\HypothesisClient\HttpClient\HttpClient;
 use eLife\HypothesisClient\HttpClient\UserAgentPrependingHttpClient;
 use GuzzleHttp\Promise\PromiseInterface;
 use GuzzleHttp\Psr7\Request;
-use Psr\Http\Message\StreamInterface;
 use Psr\Http\Message\UriInterface;
 
-trait ApiClient
+trait ApiClientTrait
```

Comment:
Since this is currently sharing the same name as something else, think about what the trait is for.

Reply:
I'm just going to get rid of the marker interface.